pytest.ini

```diff
@@ -54,5 +57,8 @@ python_files =
     # Ref: https://docs.pytest.org/en/latest/reference.html#confval-python_files
     # Needed to discover legacy nose test modules:
     Test*.py
-testpaths = test/
+    # Needed to discover embedded Rule tests
+    *Rule.py
+# do not use testpaths due to https://github.com/pytest-dev/pytest/issues/6451
```

Comment:
This link does not really explain the motivation. I think this comment should be updated.